### DISCUSSION TITLE
With 24.7.0, MTB 2D map now default displays to Sierre, Suisse

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1386] [General]
The 2D map on startup now default displays to Sierre, Suisse
The 2.5D and 3D maps all default to the most recent tour.
I have to click the show entire tour button to get the map to synchronize.  After which the 2D map works as expected.
I don't think I changed any settings but this started with the 24.7.0 release.

**Comment by Thompson-ongithub** [comment]
I just checked with my installation (24.7.0 on Linux). It defaults to my home location (not Sierre).
I suppose there's a setting somewhere to set and show a default location (when a tour without gps data is selected) but I couldn't find that right now...

**Update**:
I found the setting:
right-click somewhere in the map and select _Set Current Position as Map Default_

**Reply by stvzldn** [reply]
Right click on map, set current position as map default from menu.
This will start the 2D map in the "normal" location.
I think this was reset somehow during the update or user inadvertence.

**Comment by FJBDev** [comment]
>I don't think I changed any settings but this started with the 24.7.0 release.

While I have not experienced this issue, it seems to me that every time I install a new version, I lose all my list of recent tags that i usually find in the tour's contextual menu. does that happen to other user too ?

**Reply by wolfgang-ch** [reply]
> While I have not experienced this issue, it seems to me that every time I install a new version, I lose all my list of recent tags that i usually find in the tour's contextual menu. does that happen to other user too ?

I've not this issue, I just updated my production to 24.8 test version and the recent tags are still available

**Reply by Thompson-ongithub** [reply]
I don't remember having this issue either...
